src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -96,6 +96,16 @@ public void timColumnEncodingTest() {
     assertEquals(expectedSql, actualSql);
   }
 
+  @Test
+  public void testEnumerateSnapshotSearch() {
+    UUID testId = UUID.randomUUID();
+    List snapshotIds = List.of(testId);
```

Comment:
Our code should avoid using generic types without arguments. You can use `var` to avoid typing the arguments, if you don't want to write `List<UUID>`. (In cases where the type is unknown you should use `List<?>`, not `List`.)